h/app.py

```diff
@@ -29,6 +29,9 @@ def create_app(settings):
         api_view = wsgiapp2(api_app)
         config.add_view(api_view, name='api')
 
+    if config.registry.feature('claim'):
```

Comment:
If that's the case, then I retract my statement about moving this inside the accounts package.